relay_messages_test.go

```diff
@@ -206,25 +206,25 @@ func TestLazyCallReqRejectsOtherFrames(t *testing.T) {
 func TestLazyCallReqService(t *testing.T) {
 	withLazyCallReqCombinations(func(crt testCallReq) {
 		cr := crt.req()
-		assert.Equal(t, "bankmoji", cr.Service(), "Service name mismatch")
+		assert.Equal(t, []byte("bankmoji"), cr.Service(), "Service name mismatch")
```

Comment:
Perhaps we can cast `cr.Service()` to a string instead? I don't find the testify output from failed assertions on byte slices very helpful - strings are much easier.

Same below.